airbyte-ci/connectors/pipelines/tests/test_builds/test_python_connectors.py

```diff
@@ -31,9 +33,17 @@ def test_context_with_connector_without_base_image(self, test_context):
     def connector_with_base_image(self, all_connectors):
```

Comment:
Or it might be worth having both of these as separate fixtures. E.g. if I add something to the base image and want to assert that all connectors (regardless of customization) have it, I might want this. If I am changing some behavior that build customization might affect, I want to get the list of non-customized connectors?